docs/concepts/deployments.md

```diff
@@ -42,21 +40,24 @@ When creating a deployment, a user must answer *two* basic questions:
 A deployment additionally enables you to:
 
 - Schedule flow runs
-- Assign tags for filtering flow runs on work queues and in the Prefect UI
+- Assign `--work-queue` name to delegate flow runs to work queues
```

Comment:
This list reads like CLI documentation, I think these should only say the concept names.  For example, "Assign a work queue name to delegate..." and "Assign one or multiple tags to organize..."